pennylane/qchem/vibrational/vibrational_class.py

```diff
@@ -0,0 +1,116 @@
+# Copyright 2018-2024 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This module contains functions and classes to generate a pes object.
+This object stores all the necessary information to construct
+vibrational Hamiltonian for a given molecule."""
+
+import numpy as np
+
+import pennylane as qml
+
+from ..openfermion_pyscf import _import_pyscf
+
+# pylint: disable=import-outside-toplevel, unused-variable
+
+BOHR_TO_ANG = 0.529177
+
+
+def harmonic_analysis(scf_result, method="rhf"):
+    r"""Performs harmonic analysis by evaluating the Hessian using PySCF routines.
+    Args:
+       scf_result: pyscf object from electronic structure calculations
+       method: Electronic structure method to define the level of theory
+            for harmonic analysis. Default is restricted Hartree-Fock 'rhf'.
+
+    Returns:
+       pyscf object containing information about the harmonic analysis
+    """
+    pyscf = _import_pyscf()
+    from pyscf.hessian import thermo
+
+    method = method.strip().lower()
+    if method not in ["rhf", "uhf"]:
+        raise ValueError(f"Specified electronic structure method, {method} is not available.")
+
+    hess = getattr(pyscf.hessian, method).Hessian(scf_result).kernel()
+
+    harmonic_res = thermo.harmonic_analysis(scf_result.mol, hess)
+    return harmonic_res
+
+
+def single_point(molecule, method="rhf"):
```

Comment:
This one is also rendering differently.
![image](https://github.com/user-attachments/assets/2ca05e0f-f02e-483b-b5a0-d8aa96346c93)

Reply:
Fixed this.